docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -174,6 +174,26 @@ token can add authenticated nodes to your cluster.  These tokens can be listed,
 created and deleted with the `kubeadm token` command.  See the [reference
 guide](/docs/admin/kubeadm/#manage-tokens).
 
+#### Configure Kubectl
+
+`kubeadm init` generates several configuration files in directory /etc/kubernetes. 
```

Comment:
`kubeadm init` generates a superuser privilege kubeconfig file in `/etc/kubernetes/admin.conf`.
You can use this kubeconfig file to talk to the cluster, to do that, paste the last three commands of the `kubeadm init` output: